src/components/Pressable/GenericPressable/index.native.js

```diff
@@ -0,0 +1,20 @@
+import React, {forwardRef} from 'react';
+import GenericPressable from './BaseGenericPressable';
+import GenericPressablePropTypes from './PropTypes';
+
+const NativeGenericPressable = forwardRef((props, ref) => (
+    <GenericPressable
+        focusable
+        accessible
```

Comment:
Will these always be true? Is there ever a reason to make them false?

Reply:
I wanted them to be true by defualt, as IMO buttons should be always accessible. I'm not sure if there would be a reason to make them false, but I left the possibility to overwrite these two by passing the props `accessible={false}` or `focusable={false}`